drv/monorail-api/src/lib.rs

```diff
@@ -39,6 +39,7 @@ pub struct PortCounters {
 pub enum MonorailError {
     SpiError,
     ServerDied,
+    ProxyError,
```

Comment:
Looking for input on what this should be. `ProxyError` was the best I could think of.